src/ts/bitflip.ts

```diff
@@ -12,14 +12,17 @@ import BN from "bn.js";
  * does not matter).
  */
 export class BitFlip extends ReplayProtectionAuthority {
-  private indexTracker: Map<string, BigNumber>; // Keep track of bitmap index
-  private bitmapTracker: Map<string, BigNumber>; // Keep track of bitmap
+  private index: BigNumber; // Keep track of bitmap index
+  private bitmap: BigNumber;
   lock: Lock;
```

Comment:
I made comments on multinonce about this lock

Reply:
k